addons/website_slides/static/src/js/slides_course_quiz.js

```diff
@@ -354,6 +358,11 @@
             if (data.error) {
                 this._showErrorMessage(data.error);
                 return;
+            }
+            else if (data.wrong_anser && !data.error){
+                $('div.additional_resource_check').removeClass('d-none');
+                this._hideErrorMessage();
+                return;
             } else {
                 this._hideErrorMessage();
```

Comment:
Maybe we also should hide the links back after successful submission? (They're still in the side bar). @Francois-frgi ?